sinon/index.d.ts

```diff
@@ -109,6 +109,7 @@ declare namespace Sinon {
         callsArgOnAsync(index: number, context: any): SinonStub;
         callsArgWithAsync(index: number, ...args: any[]): SinonStub;
         callsArgOnWithAsync(index: number, context: any, ...args: any[]): SinonStub;
+        callsFake(...args: any[]): SinonStub;
```

Comment:
I don't know if this is what I meant - does `callsFake` actually take multiple arguments? I figured that it was

```ts
callsFake(func: (...args: any[]) => void)
```